llvm/runtimes/CMakeLists.txt

```
@@ -320,6 +320,10 @@ function(runtime_register_target name)
     set(install-${runtime_name}-${name} install-${runtime_name})
     set(install-${runtime_name}-${name}-stripped install-${runtime_name}-stripped)
     list(APPEND ${name}_extra_targets ${runtime_name}-${name} install-${runtime_name}-${name} install-${runtime_name}-${name}-stripped)
+    # We need the target 'libomp-mod' to run flang-rt OpenMP tests.
```

```suggestion
    # We need the target 'libomp-mod' to run flang OpenMP tests.
```
Only flang tests OpenMP, not flang-rt